plugins/modules/packaging/os/pacman.py

```diff
@@ -104,6 +104,21 @@
         default:
         type: str
 
+    reason:
+        description:
+            - The install reason to set for the packages
+        default: explicit
```

Comment:
I think it's better to not have a default, and not set/update a reason if this isn't set.